homework3/task1.py

```diff
@@ -0,0 +1,2 @@
+i = str ("i love python ")
```

Comment:
nit: there is no need to use space before the bracket
```suggestion
i = str("i love python ")
```